Makefile

```diff
@@ -128,7 +128,7 @@ fetch-release-registry:
 
 run-acceptance:
 	@echo "--> Running acceptance tests"
-	py.test tests/acceptance --cov . --cov-report="xml:.artifacts/acceptance.coverage.xml" --junit-xml=".artifacts/acceptance.junit.xml" --html=".artifacts/acceptance.pytest.html" --self-contained-html
```

Comment:
@billyvg did we need this for anything?

Reply:
Yeah at once point we uploaded this to zeus, we no longer upload anything to zeus. It's useful when acceptance shits the bed in CI (and not locally) and we have to debug, but I don't think it's something we use consistently